google/cloud/optimization/quickstart/quickstart.cc

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/optimization/fleet_routing_client.h"
+#include "google/cloud/common_options.h"
+#include "google/cloud/project.h"
+#include <google/protobuf/text_format.h>
+#include <iostream>
+#include <stdexcept>
+
+google::cloud::optimization::v1::OptimizeToursRequest LoadExampleModel();
+
+int main(int argc, char* argv[]) try {
+  if (argc != 2) {
+    std::cerr << "Usage: " << argv[0] << " project-id\n";
+    return 1;
+  }
+
+  namespace gc = ::google::cloud;
+  namespace optimization = ::google::cloud::optimization;
+
+  auto options = gc::Options{}.set<gc::UserProjectOption>(argv[1]);
+  auto client = optimization::FleetRoutingClient(
+      optimization::MakeFleetRoutingConnection(options));
+
+  auto req = LoadExampleModel();
+  req.set_parent(gc::Project(argv[1]).FullName());
+
+  auto resp = client.OptimizeTours(req);
+  if (!resp) throw std::runtime_error(resp.status().message());
+  std::cout << resp->DebugString() << "\n";
+
+  return 0;
+} catch (std::exception const& ex) {
+  std::cerr << "Standard exception raised: " << ex.what() << "\n";
+  return 1;
+}
+
+/**
```

Comment:
I wonder if we could build a quickstart around this data:

```
gsutil ls gs://cloud-samples-data/optimization-ai
```

and

https://cloud.google.com/optimization/docs/reference/rpc/google.cloud.optimization.v1#batchoptimizetoursrequest

which after several indirections allows you to use a file from GCS?